keyboards/sevencrumbs/matrix/info.json

```diff
@@ -0,0 +1,44 @@
+{
+	"keyboard_name": "Matrix",
+	"manufacturer": "Seven Crumbs",
+	"url": "https://zoid.com.au/matrix/",
+	"maintainer": "ZoidTechnology",
+	"bootloader_instructions": "Press and hold all four buttons while connecting the USB cable.",
+	"processor": "RP2040",
+	"board": "GENERIC_RP_RP2040",
+	"bootloader": "rp2040",
+	"bootmagic": {
+		"enabled": true
+	},
```

Comment:
```suggestion
    "features": {
        "bootmagic": true,
        "mousekey": true,
        "extrakey": true,
        "rgb_matrix": true
    },
```

Reply:
Great suggestion! I didn't know this was possible.